package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "dotenv": "^8.1.0",
+    "mcrypt": "^0.1.17",
```

Comment:
As I see `mcrypt` package has been deprecated and not maintained anymore. The author suggests using `cryptian` instead

Reply:
mcrypt was the only decoder that shows the strange characters after } in the json returning from the machine.